administration/src/components/user-settings/UserSettingsController.tsx

```diff
@@ -52,18 +55,22 @@ const UserSettingsController = () => {
   const valid = warnMessage === null
 
   return (
-    <ChangePasswordForm
-      currentPassword={currentPassword}
-      setCurrentPassword={setCurrentPassword}
-      newPassword={newPassword}
-      setNewPassword={setNewPassword}
-      repeatNewPassword={repeatNewPassword}
-      setRepeatNewPassword={setRepeatNewPassword}
-      submitDisabled={!valid}
-      warnMessage={isDirty ? warnMessage : null}
-      loading={loading}
-      submit={submit}
-    />
+    <>
+      <ChangePasswordForm
+        currentPassword={currentPassword}
+        setCurrentPassword={setCurrentPassword}
+        newPassword={newPassword}
+        setNewPassword={setNewPassword}
+        repeatNewPassword={repeatNewPassword}
+        setRepeatNewPassword={setRepeatNewPassword}
+        submitDisabled={!valid}
+        warnMessage={isDirty ? warnMessage : null}
+        loading={loading}
+        submit={submit}
+      />
+      {/* TODO #897: [Application] Remove Redirect for bayern */}
+      {applicationFeatureEnabled && <ApplicationLinkCard />}
```

Comment:
I think it would make more sense, if it was on the `ApplicationsOverview` tab somewhere (eg. above everything else).